test/test_binary_ufuncs.py

```diff
@@ -650,7 +650,7 @@ def to_np(value):
             for test_case in test_cases:
                 self.assertRaisesRegex(RuntimeError, err_msg, test_case)
         else:
-            if isinstance(base, torch.Tensor):
+            if isinstance(base, torch.Tensor) and base.dim() > 0:
```

Comment:
"reduction" of test coverage

Reply:
fair point. I'll update this to more explicitly check for the change I'm adding: you can't call `pow(torch.tensor(2), torch.tensor([2, 2], device="cuda"))` with the in-place variant of `pow`, since the output tensor in that case is the base, which is already on cpu.